source/MaterialXGenOsl/OslSyntax.cpp

```diff
@@ -407,10 +469,12 @@ OslSyntax::OslSyntax()
     registerTypeSyntax
     (
         Type::FILENAME,
-        std::make_shared<StringTypeSyntax>(
-            "string",
-            "\"\"",
-            "\"\"")
+        std::make_shared<FileFormatSyntax>(
+            "FILEINPUT",
+            "FILEINPUT(\"\", \"\")",
```

Comment:
Nitpick: Should the name imply that this is a file texture? Maybe naming it TEXTURE?

Reply:
I just made up a name. We can go with TEXTURE if there is no name clashing. Maybe FILETEXTURE ?